jgrapht-ext/src/main/java/org/jgrapht/ext/DIMACSExporter.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * (C) Copyright 2016-2016, by Dimitrios Michail and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+package org.jgrapht.ext;
+
+import java.io.PrintWriter;
+import java.io.Writer;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+import org.jgrapht.Graph;
+
+/**
+ * Exports a graph into DIMACS format.
+ *
+ * <p>
+ * For a description of the format see <a href="http://dimacs.rutgers.edu/Challenges/">
```

Comment:
Maybe I'm missing something, but I couldn't find a proper format description on this website? Which document did you use to implement this? I think in the past I looked at this: http://www.or.uni-bonn.de/lectures/ss12/praktikum/ccformat.pdf
for the graph coloring problem type.
Problem is, a lot of the urls on the page you reference simply don't work anymore :(

Either way, I would suggest to:
add a link to a page with a full format description
or
we can link to some pdfs. Optionally we can copy the pdf and put it in our website repository.